db/fixtures/coordinators.rb

```diff
@@ -1,4 +1,4 @@
-20.times do
+13.times do
```

Comment:
I don't remember. I was just changing numbers around to get better and more meaningful association building during seeding without making _a million_ models.